Automattic/vip-design-system
# Improvements for Focus and Disabled states

## Description

- Introducing a standard color for `outline` focused elements;
- Adding the disabled state for the Toggle (bugfix since we changed to the new implementation);
- Increasing opacity to `0.7` instead of `0.5`.

<img width="293" alt="image" src="https://user-images.githubusercontent.com/3402/180058523-76d50a49-7e23-4641-a691-06a31d397607.png">

<img width="1076" alt="image" src="https://user-images.githubusercontent.com/3402/180058563-3576d809-4129-48ca-8cc9-69b76c628b17.png">

<img width="327" alt="image" src="https://user-images.githubusercontent.com/3402/180058591-7cbd4e96-243b-401d-8d37-fc54f6edf557.png">


## Checklist


## Steps to Test

1. Pull down PR.
2. `npm run dev`.
3. Open Storybook.
4. Open the [NewDialog](http://localhost:6006/?path=/story/newdialog--default) example, and check the button outline
5. Open the [Toggle](http://localhost:6006/?path=/story/toggle--primary) example, and check there is a new example for Disabled state
6. Open the [Button](http://localhost:6006/?path=/story/button--default) example and check there's the standard outline color of all examples above
